rest/config.go

```diff
@@ -719,8 +719,16 @@ func (config *ServerConfig) deprecatedConfigLoggingFallback() (warnings []base.D
 			warnings = append(warnings, func() {
 				base.Warnf(warningMsgFmt, `logging.["default"].LogFilePath`, "logging.log_file_path")
 			})
+
 			// Set the new LogFilePath to be the directory containing the old logfile, instead of the full path.
-			config.Logging.LogFilePath = filepath.Dir(*config.Logging.DeprecatedDefaultLog.LogFilePath)
+			// SGCollect relies on this path to pick up the standard and rotated log files.
+			info, err := os.Stat(*config.Logging.DeprecatedDefaultLog.LogFilePath)
+			if err == nil && info.IsDir() {
+				config.Logging.LogFilePath = *config.Logging.DeprecatedDefaultLog.LogFilePath
+			} else {
+				base.Infof(base.KeyAll, "Using %v as log file path (parent directory of deprecated logging.["default"].LogFilePath)", config.Logging.LogFilePath)
```

Comment:
You need to move this *after* line 730.

Reply:
Sure Adam, this is moved down right after setting the path.